### PR TITLE
Add privacy notice to welcome page

### PR DESCRIPTION
**Pull request by alex-demetriou**
### Why
Users will be advised that information about their usage will be sent to Google Analytics and Hotjar when logging in.

### What
[MA-210](https://hackney.atlassian.net/browse/MA-210): Add Privacy Text to the signing in page

### CODE REVIEW

**Comment by kjdchapman** [comment]
This looks sensible to me, but can you include a screenshot of how it looks on the page?

**Comment by alex-demetriou** [comment]
> This looks sensible to me, but can you include a screenshot of how it looks on the page?

Added

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -8,5 +8,11 @@
     <%= link_to auth_provider_path do %>
       <%= image_tag 'microsoft_login.png', alt: 'Sign in with Microsoft' %>
     <% end %>
+    
+    <br><br><br>
```
Comment (richrace) [inline app/views/sessions/new.html.erb]: I'm not too keen on this. I'd rather see the privacy in a `div` and some CSS applied

@kjdchapman thoughts?

Comment (kjdchapman) [reply app/views/sessions/new.html.erb]: Yes, I agree (I feel fairly strongly about it too!)

**Pull request merged**